### PR TITLE
homework day3

### PR DESCRIPTION
**Pull request by RoelofWobben**
Everything is working except Class3.Demo still does more then 1 thing but I do not know how I can make that better without using for example a Person class.

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Class3.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846210&bid=16085227)

File: Src/BootCamp.Chapter/Class3.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Class3
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Class3.cs]: Name this class "Lesson3"

Comment (RoelofWobben) [reply Src/BootCamp.Chapter/Class3.cs]: done, is changed. I did not read the homework properly. Thanks for the remark

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4846210&bid=16085227)

**Pull request closed without merging**